versioned_docs/version-0.6.0/kubeslice-cli/getting-started.mdx

```diff
@@ -47,6 +47,32 @@ To install kubeslice-cli on Linux, macOS, or Ubuntu:
    ```
```

Comment:
This command applies to line no 46: 

up-to-date is an adjective but you are using it as a noun. It can be latest instead of up to date

Reply:
done